src/OpenSage.Game/Logic/Object/Behaviors/ParkingPlaceBehavior.cs

```diff
@@ -204,15 +204,15 @@ public void ReserveRunway(uint objectId)
         // if all runways are occupied, check if we can double-stack
         for (var i = 0; i < _runwayAssignments.Length; i++)
         {
-            if (_runwayAssignments[i].Aircraft2Id == 0)
+            if (!_runwayAssignments[i].Aircraft2Id.IsValid)
```

Comment:
Is it worth adding a convenience `Invalid` (or IsInvalid) property instead of all the negate calls? Seems like we're negating this a lot.

Reply:
Added, and also added some documentation comments for ObjectId's fields.